Patches/JDS EFT Apparel/Apparel_HeadgearArmor.xml

```diff
@@ -0,0 +1,397 @@
+            <ArmorRating_Heat>0.4</ArmorRating_Heat>
+          </value>
+        </li>
+
+        <!-- === Class 3 === -->
+        <li Class="PatchOperationAdd">
+          <xpath>
+            /Defs/ThingDef[
+              defName = "JDSTarkov_Headgear_UNTAR_Helmet" or
+              defName = "JDSTarkov_Headgear_Ratnik-BSh_Helmet" or
+              defName = "JDSTarkov_Headgear_LZSh_Light_Helmet" or
+              defName = "JDSTarkov_Headgear_SSh-68_Helmet" or
+              defName = "JDSTarkov_Headgear_Kiver_M_Helmet" or
+              defName = "JDSTarkov_Headgear_Kiver_M_Helmet_Shield" or
+              defName = "JDSTarkov_Headgear_Kiver_M_Helmet_Shieldup" or
+              defName = "JDSTarkov_Headgear_SSSh_95_Sfera_S"
+            ]/statBases
+          </xpath>
+          <value>
+            <Bulk>4</Bulk>
+            <WornBulk>1</WornBulk>
+            <ArmorRating_Sharp>5</ArmorRating_Sharp>
+            <ArmorRating_Blunt>8</ArmorRating_Blunt>
+            <ArmorRating_Heat>0.4</ArmorRating_Heat>
+          </value>
+        </li>
+
+        <!-- === Class 4 === -->
+         <li Class="PatchOperationAdd">
+          <xpath>
+            /Defs/ThingDef[
+              defName = "JDSTarkov_Headgear_MSA_TC_800_Helmet" or
+              defName = "JDSTarkov_Headgear_MSA_TC_2001_Helmet" or
+              defName = "JDSTarkov_Headgear_MSA_TC_2002_Helmet" or
+              defName = "JDSTarkov_Headgear_Highcom_Striker_ACHHC_IIIA_Helmet" or
+              defName = "JDSTarkov_Headgear_Highcom_Striker_ACHHC_IIIA_Helmet_Black" or
+              defName = "JDSTarkov_Headgear_Highcom_Striker_ULACH_IIIA_Helmet" or
+              defName = "JDSTarkov_Headgear_ZSh-1-2M_Helmet" or
+              defName = "JDSTarkov_Headgear_ZSh-1-2M_Helmet_Shield" or
+              defName = "JDSTarkov_Headgear_Ops-Core_Fast_MT_Helmet" or
+              defName = "JDSTarkov_Headgear_Ops-Core_Fast_MT_Helmet_Shield" or
+              defName = "JDSTarkov_Headgear_Ops-Core_Fast_MT_Mando" or
+              defName = "JDSTarkov_Headgear_Ops-Core_Fast_mod_MT" or
+              defName = "JDSTarkov_Headgear_Ops-Core_Fast_Full_MT" or
+              defName = "JDSTarkov_Headgear_Ops-Core_Fast_MT_Ten" or
+              defName = "JDSTarkov_Headgear_BNTI_LSHZ-2DTM_Helmet" or
+              defName = "JDSTarkov_Headgear_BNTI_LSHZ-2DTM_Helmet_Shield" or
+              defName = "JDSTarkov_Headgear_BNTI_LSHZ-2DTM_Helmet_Full" or
+              defName = "JDSTarkov_Headgear_Maska_1Sch_Helmet" or
+              defName = "JDSTarkov_Headgear_Maska_1Sch_Helmet_Shield" or
+              defName = "JDSTarkov_Headgear_Maska_1Sch_Helmet_Shieldup" or
+              defName = "JDSTarkov_Headgear_Tac-Kek_Fast_MT" or
+              defName = "JDSTarkov_Headgear_Tac-Kek_Fast_MT_Mando"
+            ]/statBases
+          </xpath>
+          <value>
+            <Bulk>3</Bulk>
+            <WornBulk>1</WornBulk>
+            <ArmorRating_Sharp>6</ArmorRating_Sharp>
```

Comment:
I  just find that the stats of class 4 are identical to class 5. maybe need some changes?

Reply:
Actually was waiting for ideas here since i'm not sure what numbers to use here